watchtower/wtdb/client_db.go

```diff
@@ -20,8 +20,13 @@ var (
 
 	// cChanDetailsBkt is a top-level bucket storing:
 	//   channel-id => cChannelSummary -> encoded ClientChanSummary.
+	//  		=> cChanDBID -> db-assigned-id
```

Comment:
Instead of roll a new 8 byte value to "compress" a channel ID, what if we just used the existing short channel ID value? This is assumed to be unique from the PoV of the main chain.

Reply:
went with this cause we register a channel with the client at the point of the link starting and with zero-conf, we dont necessarily know the SCID at this point so thought it was best to keep things simple in that regard.

Reply:
@bhandras: so I just did a quick test with the same 3GB wtclient db & found that with the current code (using the BigSize db assigned chanID) the resulting db (after migration and compaction) was 185MB. (16x gain)
And with just using the channel ID as is, the resulting DB was 818MB (3.7x gain)

Reply:
I think that's very much in the the worth it territory!